### PR TITLE
Remove criu spec suffix

### PR DESCRIPTION
**Pull request by llxia**
Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
[CRIU test build](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_x86-64_linux_criu_Personal/1/) got triggered with the wrong PLATFORM value - `x86-64_linux_criu`. This PR is to remove `_criu` suffix in PLATFORM.

```
java.lang.AssertionError: Cannot find key PLATFORM: x86-64_linux_criu in PLATFORM_MAP: [arm_linux:[SPEC:linux_arm, LABEL:ci.role.test&&sw.os.linux&&hw.arch.aarch32], aarch64_mac:[SPEC:osx_aarch64, LABEL:ci.role.test&&hw.arch.aarch64&&(sw.os.osx||sw.os.mac)], aarch64_linux:[SPEC:linux_aarch64, LABEL:ci.role.test&&sw.os.linux&&hw.arch.aarch64], ppc32_aix:[SPEC:aix_ppc, LABEL:ci.role.test&&hw.arch.ppc64&&sw.os.aix], ppc32_linux:[SPEC:linux_ppc, LABEL:ci.role.test&&hw.arch.ppc64&&sw.os.linux], ppc64_aix:[SPEC:aix_ppc-64, LABEL:ci.role.test&&hw.arch.ppc64&&sw.os.aix], ppc64_linux:[SPEC:linux_ppc-64, LABEL:ci.role.test&&hw.arch.ppc64&&sw.os.linux], ppc64le_linux:[SPEC:linux_ppc-64_le, LABEL:ci.role.test&&hw.arch.ppc64le&&sw.os.linux, DynamicAgents:[fyre]], riscv64_linux:[SPEC:linux_riscv64, LABEL:ci.role.test&&sw.os.linux&&hw.arch.riscv&&hw.bits.64], riscv64_linux_xl:[SPEC:linux_riscv64, LABEL:ci.role.test&&sw.os.linux&&hw.arch.riscv&&hw.bits.64], s390_linux:[SPEC:linux_390, LABEL:ci.role.test&&hw.arch.s390x&&sw.os.linux&&hw.bits.32], s390x_linux_zt:[SPEC:linux_390-64_zt, LABEL:ci.role.test&&hw.arch.znext&&sw.os.linux], s390x_linux:[SPEC:linux_390-64, LABEL:ci.role.test&&hw.arch.s390x&&sw.os.linux], s390x_zos:[SPEC:zos_390-64, LABEL:ci.role.test&&hw.arch.s390x&&sw.os.zos], s390x_zos_xl:[SPEC:zos_390-64, LABEL:ci.role.test&&hw.arch.s390x&&sw.os.zos], s390x_zos_zt:[SPEC:zos_390-64_zt, LABEL:ci.role.test&&hw.arch.znext&&sw.os.zos], s390_zos_zt:[SPEC:zos_390_zt, LABEL:ci.role.test&&hw.arch.znext&&sw.os.zos], s390_zos:[SPEC:zos_390, LABEL:ci.role.test&&hw.arch.s390x&&sw.os.zos], sparcv9_solaris:[SPEC:sunos_sparcv9-64, LABEL:ci.role.test&&hw.arch.sparcv9&&sw.os.sunos], x86-64_solaris:[SPEC:sunos_x86-64, LABEL:ci.role.test&&hw.arch.x86&&sw.os.sunos], x86-64_alpine-linux:[SPEC:alpine-linux_x86-64, LABEL:ci.role.test&&hw.arch.x86&&sw.os.alpine-linux], x86-32_linux:[SPEC:linux_x86, LABEL:ci.role.test&&hw.arch.x86&&sw.os.linux], x86-32_windows:[SPEC:win_x86, LABEL:ci.role.test&&hw.arch.x86&&sw.os.windows], x86-64_linux:[SPEC:linux_x86-64, LABEL:ci.role.test&&hw.arch.x86&&sw.os.linux, DynamicAgents:[azure, fyre]], x86-64_mac:[SPEC:osx_x86-64, LABEL:ci.role.test&&hw.arch.x86&&(sw.os.osx||sw.os.mac)], x86-64_windows:[SPEC:win_x86-64, LABEL:ci.role.test&&hw.arch.x86&&sw.os.windows]].. Expression: assert false : "Cannot find key PLATFORM: ${params.PLATFORM} in PLATFORM_MAP: ${PLATFORM_MAP}."
```

**Comment by AdamBrousseau** [comment]
Jenkins test sanity xlinuxcriu jdk11

File: buildenv/jenkins/common/pipeline-functions.groovy [hunk 1]
```diff
@@ -806,7 +806,7 @@ def move_spec_suffix_to_id(spec, id) {
     def spec_id = [:]
     spec_id['spec'] = spec
     spec_id['id'] = id
-    for (suffix in ['aot', 'cm', 'jit', 'valhalla', 'uma', 'ojdk292', 'vt_standard']) {
+    for (suffix in ['aot', 'cm', 'jit', 'valhalla', 'uma', 'ojdk292', 'vt_standard', 'criu']) {
```
Comment (pshipton) [inline buildenv/jenkins/common/pipeline-functions.groovy:809]: Pls put the whole list in alphabetical order

Comment (llxia) [reply buildenv/jenkins/common/pipeline-functions.groovy]: updated

File: buildenv/jenkins/common/pipeline-functions.groovy [hunk 2]
```diff
@@ -806,7 +806,7 @@ def move_spec_suffix_to_id(spec, id) {
     def spec_id = [:]
     spec_id['spec'] = spec
     spec_id['id'] = id
-    for (suffix in ['aot', 'cm', 'jit', 'valhalla', 'uma', 'ojdk292', 'vt_standard']) {
+    for (suffix in ['aot', 'cm', 'criu', 'jit', 'valhalla', 'uma', 'ojdk292', 'vt_standard']) {
```
Comment (pshipton) [inline buildenv/jenkins/common/pipeline-functions.groovy:809]: All the entries can be arranged alphabetically.
'aot', 'cm', 'criu', 'jit', 'ojdk292', 'uma', 'valhalla', 'vt_standard'

**Comment by llxia** [comment]
The test build is triggered with the correct platform: https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_x86-64_linux_criu_Personal/4/rebuild/parameterized

image

**Comment by llxia** [comment]
FYI @tajila

**Review by pshipton — APPROVED** [review]

**Review by tajila — APPROVED** [review]

**Comment by tajila** [comment]
Jenkins test sanity xlinuxcriu jdk11

**Review by AdamBrousseau — APPROVED** [review]

**Pull request merged**